packages/bpk-docs/src/pages/MapPage/MapPage.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import BpkMap, { withScriptjs } from 'bpk-component-map';
+
+import mapReadme from 'bpk-component-map/readme.md';
+
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import DocsPageWrapper from './../../components/neo/DocsPageWrapper';
+import IntroBlurb from './../../components/neo/IntroBlurb';
+
+const BpkMapWithScript = withScriptjs(BpkMap);
+const MAP_URL =
+  'https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places';
```

Comment:
What kind of usage limits are there for unauthenticated requests to Google Maps?

Reply:
We should be alright hopefully: https://developers.google.com/maps/documentation/javascript/usage. Looks like 25,000 map loads per 24 hours. If this becomes an issue, we can always get Skyscanner's API key.